src/foam/nanos/crunch/crunchtest/CapabilityTest.java

```diff
@@ -80,27 +73,31 @@ public void runTest(X x) {
     testCapability(x);
   }
 
+  // TODO: Actually test something here.
```

Comment:
This comment could be more specific so the call to `test` below doesn't cause someone to mistakenly think it's outdated.